tools/serverpod_cli/lib/src/generator/dart/library_generators/library_generator.dart

```diff
@@ -837,6 +839,15 @@ class LibraryGenerator {
     return methodConnectors;
   }
 
+  String? _getMethodCallComment(MethodCallDefinition m) {
```

Comment:
Suggestion: Please add a test for the library generator to validate that the comment is inserted as expected when supplied.